frontend/src/views/ProjectView.vue

```diff
@@ -149,13 +157,29 @@ function updateIsActive(newValue: boolean) {
 }
 
 async function getAllFiles(projectId: string | null) {
-  var result: File[] = []
+  var result: FileClass[] = []
   if (projectId == null) {
     return result
   }
   try {
     const reponse = await storeApp.axiosInstance.get<File[]>(`/projects/${projectId}/files`)
-    result = reponse.data
+    for (const file of reponse.data) {
+      result.push(
+        new FileClass(
```

Comment:
I think a dedicated method on NautilusFile that takes a `File` as param would be more readable/suimpler to use:

```js
result.push(NautilusFile.fromFile(file))
```